.github/workflows/main.yml

```diff
@@ -39,6 +39,8 @@ jobs:
       - name: Build and push
         id: docker_build
         uses: docker/build-push-action@v2
+        env: 
+          NODE_OPTIONS: --max-old-space-size=8192
```

Comment:
this only fixes the github test/build
shouldn't we add the same node options to docker build?